packages/iris-grid/src/formatters/DateTimeColumnFormatter.ts

```diff
@@ -1,17 +1,19 @@
 /* eslint class-methods-use-this: "off" */
 import dh from '@deephaven/jsapi-shim';
 import Log from '@deephaven/log';
-import TableColumnFormatter from './TableColumnFormatter';
+import TableColumnFormatter, {
+  TableColumnFormat,
```

Comment:
Just FYI I looked into the `import type` more and what it does is enforces that something that isn't a type can only be used for its types within the file.

If you use a regular import for a class and then only use it for its types, it will be stripped from the compiled JS import. So I guess import type probably isn't needed other than to be explicit. Pretty sure that previous comment I had about `import type` would result in the same code regardless of the import type.

Basically import type should help if there's a build failure due to the isolated modules flag and other than that it's just us being explicit. Isolated modules flag will cause an error if there's some imports that. I think this example is an example of that flag causing a failure: https://github.com/microsoft/TypeScript/issues/28481#issuecomment-441284387

Basically re-exporting types is problematic for Babel because it only looks at a single file. So it has no idea if the re-exported value is a type or a value (I think it errs on considering it a value). That can result in unnecessary/broken imports in the final code

https://github.com/microsoft/TypeScript/pull/35200#issuecomment-645565073